src/components/map.js

```diff
@@ -5,7 +5,7 @@ import Search from './search';
 import Response from '../components/response';
 
 let MapboxGl = null;
-const apiCallNullMessage = 'http://159.89.238.220/v1/autocomplete?text='
+const apiCallNullMessage = 'http://159.89.238.220/v2/autocomplete?text='
```

Comment:
Hi @chriswhong! Thanks for opening this PR. Updating this docs site is definitely on our radar and this PR should make a great start. Do you happen to know what this IP address is meant to point to and do you think maybe it should be updating to point to the planninglabs.nyc URL like you did in `search.js`? If that IP is no longer relevant, I think we should remove references to it. Other than that, this PR lgtm.

Reply:
Hey there! Yet another Planning Labs alum here 🙂 [I use this API in a class I teach](https://github.com/afeld/python-public-policy/blob/562da401867fc5e46646bf61e2ab66fe4e8cc18d/hw_5_old.ipynb), and with that lecture coming up in a few weeks, I worry the students (who are new to APIs) will be confused if the documentation is out of date. Since it's not a regression, could these changes be merged sooner than later and the IP address question handled as a follow-up? 🙏

Reply:
Hi, sorry but I have no idea what that hard-coded IP address was about.  Probably wise to remove all references to it.